source/includes/harvest/_demographic_data.md

```diff
@@ -0,0 +1,463 @@
+# Demographic Data
+
+Greenhouse Inclusion ...
```

Comment:
Demographic questions and answers submitted during the application process. For more information on Greenhouse Inclusion, please visit https://www.greenhouse.io/inclusion.

Reply:
Thanks @mmurph24!